assets/react/v3/shared/components/modals/ModalWrapper.tsx

```diff
@@ -37,25 +67,63 @@ export default ModalWrapper;
 
 const styles = {
   container: css`
+    position: relative;
     background: ${colorTokens.background.white};
     margin: ${spacing[24]};
-    max-width: 1236px;
+    margin-top: 60px;
+    height: 100%;
+    max-width: 1472px;
     box-shadow: ${shadow.modal};
     border-radius: ${borderRadius[10]};
     overflow: hidden;
+    bottom: 0;
 
     ${Breakpoint.smallTablet} {
       width: 90%;
     }
   `,
   header: css`
-    display: flex;
+    display: inline-flex;
     justify-content: space-between;
     align-items: center;
     padding: ${spacing[20]};
     width: 100%;
+    max-height: 72px;
     background: ${colorTokens.background.white};
     border-bottom: 1px solid ${colorTokens.stroke.divider};
+    position: sticky;
+  `,
+  headerContent: css`
+    display: inline-flex;
+    align-items: center;
+    gap: ${spacing[12]};
+
+    & span {
+      ::before {
+        content: '';
+        border: 0.5px solid ${colorTokens.icon.hints};
```

Comment:
Fraction px value may not work all the browsers well. Use nearest integer value.